src/components/shoppingCart/cartdetail/CartDetail.jsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import { MOCK_DATABASE } from "../../common/MockDatabase";
+
+const { title, images, brand, price } = MOCK_DATABASE[0];
+
+const CartDetail = () => {
+  const cartDetailHeaderRow = (
+    <Row className="cartDetailHeader">
+      <Col className="cartDetailTitle">Shopping Card (X Items)</Col>
+      <Col className="backToShoppingWrapper">
+        <Button className="backToShopping">Back to Shopping!</Button>
+      </Col>
+    </Row>
+  );
+
+  const titleHeaderRow = (
+    <Row className="titleHeader">
+      <Col
+        xl={6}
```

Comment:
if all of these are the same im pretty sure you dont have to write it 5 times, probably only one of these will encompass all

Reply:
No, it seems not working that way. But the documentation says if we want the rest to auto divide the space, we do not explicitly define them so I did it so that it's siblings are gonna inherit the rest of it:

[https://react-bootstrap.github.io/layout/grid/#auto-layout-col-sizing](url)